test/Interop/Cxx/class/access/access-inversion-typechecker.swift

```diff
@@ -276,16 +276,15 @@ func usePrivateEnumClass(a: inout Leaky.AliasToPrivateEnumClass) -> Leaky.AliasT
     // Constructing and reading PrivateEnumClass
     //
 
-    // NOTE: private enum class members are not accessible even if we can access
-    // instances of the private enum class via
+    // NOTE: private enum class members are accessible if we can access
+    // their parent enum class decl
 
     let _ = Leaky.AliasToPrivateEnumClass.privateEnumClassMember
-    // expected-error@-1 {{'privateEnumClassMember' is inaccessible due to 'private' protection level}}
     let _ = Leaky.PrivateEnumClass.privateEnumClassMember
     // expected-error@-1 {{'PrivateEnumClass' is inaccessible due to 'private' protection level}}
     let _: Leaky.AliasToPrivateEnum = .privateEnumClassMember
     // expected-error@-1 {{type 'Leaky.AliasToPrivateEnum' (aka 'Leaky.PrivateEnum') has no member 'privateEnumClassMember'}}
-    // TODO: ^this is not really the right error message
+    // FIXME: ^this should be accessible
```

Comment:
Isn't privateEnumClassMember an element in PrivateEnumClass, not PrivateEnum? I don't see what's wrong here

Reply:
Oh. Good catch!